frontend/simple/views/NavBar.vue

```diff
@@ -1,160 +1,134 @@
+        >
+          <i18n>Start a Group</i18n>
+        </router-link>
+
+        <router-link
+          active-class="is-active"
+          class="navbar-item gi-item"
+          to="pay-group"
+        >
+          <i18n>Pay Group</i18n>
+        </router-link>
+
+        <router-link
+          active-class ="is-active"
+          class="navbar-item gi-item"
+          id="MailboxLink"
+          to="mailbox"
+        >
+          <i18n>Inbox</i18n>
+          <span
+            id="AlertNotification"
+            class="icon"
+            style="color: #ed6c63;"
+            v-if="$store.getters.unreadMessageCount || $store.getters.proposals.length"
           >
-            <i18n>Start a Group</i18n>
-          </router-link>
+            <i class="fa fa-bell"></i>
+          </span>
+        </router-link>
+      </div>
+      <div class="navbar-end is-flex">
+        <div class="navbar-item signUp-item" v-if="!$store.state.loggedIn">
           <router-link
-            active-class="is-active"
-            class="nav-item is-tab"
-            to="pay-group"
-            v-if="$store.state.loggedIn"
+            class="button is-success"
+            id="SignupBtn"
+            to="signup"
           >
-            <i18n>Pay Group</i18n>
+            <i18n>Sign Up</i18n>
           </router-link>
-          <router-link
-            active-class ="is-active"
-            class="nav-item"
-            id="MailboxLink"
-            to="mailbox"
-            v-if="$store.state.loggedIn"
+        </div>
+        <div class="navbar-item" v-if="!$store.state.loggedIn">
+          <button class="button is-primary"
+            id="LoginBtn"
+            @click="showLoginModal"
           >
-            <i18n>Inbox</i18n>
-            <span
-              id="AlertNotification"
-              class="icon"
-              style="color: #ed6c63;"
-              v-if="$store.getters.unreadMessageCount || $store.getters.proposals.length"
-            >
-              <i class="fa fa-bell"></i>
-            </span>
-          </router-link>
+            <i18n>Login</i18n>
+          </button>
         </div>
-        <div class="nav-right">
-          <span class="nav-item control">
-            <router-link
-              class="button is-success"
-              id="SignupBtn"
-              style="margin-right: 15px;"
-              to="signup"
-              v-if="!$store.state.loggedIn"
+        <div id="OpenProfileDropDown"
+          class="navbar-item has-dropdown is-hoverable gi-is-profile"
+          v-if="$store.state.loggedIn">
+          <a class="navbar-link">
+            <strong>
+              {{ ($store.getters.currentUserIdentityContract &&
+                $store.getters.currentUserIdentityContract.attributes &&
+                $store.getters.currentUserIdentityContract.attributes.displayName ?
+                  $store.getters.currentUserIdentityContract.attributes.displayName :
+                  null) ||
```

Comment:
I'm pretty sure this can be simplified to:

```js
($store.getters.currentUserIdentityContract &&
$store.getters.currentUserIdentityContract.attributes &&
$store.getters.currentUserIdentityContract.attributes.displayName) || $store.state.loggedIn.name
```

You don't need the ternary operator, because if it returns `null` then the `||` will still kick in and the result of the entire expression will be `$store.state.loggedIn.name`

Reply:
I've only updated the indentation because of the `<div>` removed, without touching too much on the logic part to keep the focus only on Bulma's update. But Git isn't smart enough to recognize that part, so it thinks I've wrote that. Nevertheless, I completely agree with your comment.

Reply:
@sandrina-p OK, roger, however, I don't remember the word that @hubudibu used for it, but, it's the idea of "as you notice things (if you notice them), feel free to clean up code if it's not too much  trouble". I don't remember the name she came up with for this concept, but I like it a lot :)

Reply:
I agree with you but our Contributing guidelines contradicts that
> A PR SHOULD be a minimal and accurate answer to exactly one identified and agreed problem.

That's why I've update Bulma with a mindset of not touching anything besides Bulma.

Maybe we should add @hubudibu's word/mindset/expression to our guidelines, so we can all be more flexible. 🙌

Reply:
Great idea 👍

Reply:
Refactor done